### PR TITLE
Fix flaky test

### PR DESCRIPTION
**Pull request by destijl**

### CODE REVIEW

**Comment by destijl** [comment]
@joachimmetz

File: artifacts/registry.py [hunk 1]
```diff
@@ -49,8 +49,7 @@ def GetDefinitions(self):
     Yields:
```
Comment (joachimmetz) [inline artifacts/registry.py]: change the docstring accordingly

**Comment by joachimmetz** [comment]
LGTM, one small docstring nit

File: artifacts/registry.py (hunk 1 above)
Comment (destijl) [inline artifacts/registry.py]: done.

**Pull request merged**